src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -250,10 +250,7 @@ extends AbstractSeq[T] with IndexedSeq[T] with Serializable {
       super.equals(other)
   }
 
-  override def toString() = {
-    val endStr = if (length > Range.MAX_PRINT) ", ... )" else ")"
-    take(Range.MAX_PRINT).mkString("NumericRange(", ", ", endStr)
-  }
+  override def toString = "%d %s %d by %s".format(start, if (isInclusive) "to" else "until", end, step)
```

Comment:
I wouldn't even use the `f` formatter. Just use `s`. Let the numbers display themselves as it pleases them.